sdk/tables/azure-data-tables/README.md

```diff
@@ -92,6 +93,30 @@ connection_string = "DefaultEndpointsProtocol=https;AccountName=<my_account_name
 service = TableServiceClient.from_connection_string(conn_str=connection_string)
 ```
 
+```python
+from azure.data.tables import TableServiceClient
+connection_string = "DefaultEndpointsProtocol=https;AccountName=<my_account_name>;AccountKey=<my_account_key>;TableEndpoint=https://<my_account_name>.table.cosmos.azure.com"
+service = TableServiceClient.from_connection_string(conn_str=connection_string)
+```
+
+For Tables Cosmos, the connection string can be found in your cosmos account in the [Azure Portal][azure_portal_account_url] under the "Connection Strings" section or with the following Azure CLI command:
+
+```bash
+az cosmosdb list-connection-strings -g MyResourceGroup -n MyCosmosAccount
+```
+
+```python
+from azure.data.tables import TableServiceClient
+connection_string = "DefaultEndpointsProtocol=https;AccountName=<my_account_name>;AccountKey=<my_account_key>;EndpointSuffix=cosmos.azure.com"
+service = TableServiceClient.from_connection_string(conn_str=connection_string)
+```
+
+```python
+from azure.data.tables import TableServiceClient
+connection_string = "DefaultEndpointsProtocol=https;AccountName=<my_account_name>;AccountKey=<my_account_key>;TableEndpoint=https://<my_account_name>.table.cosmos.azure.com"
```

Comment:
I don't think we need 3 different examples of a Cosmos connection string - I think just the first one is fine.

Reply:
Updated, just have one example now.